docs/platform/concepts/trial-project-features.rst

```diff
@@ -0,0 +1,34 @@
+Trial project features
+======================
+
+Aiven platform offers a free 30 day trial to explore the platform along with many managed Aiven service offerings, including: 
+
+* Apache Cassandra®
+* Apache Kafka®
+* ClickHouse®
+* Grafana®
+* InfluxDB®
+* MySQL
+* OpenSearch®
+* PostgreSQL®
+* Redis®*
+
+
+.. note:: **A credit card is not required to start a trial.** 
+ 
+When you sign up on Aiven for the first time, a default (trial) project will be created. The trial grants full access to services, the management console, and other access including the REST API, CLI tool and Terraform provider. 
+
+**The trial project has some limitations:**
+
+* Auto-generated trial projects cannot be renamed
+* Trial credits cannot be transferred to another project
+* A Maximum of **10** VMs can be created
+* A Maximum of **1** VPC and **10** VPC peering connections
+
+.. important:: Trial projects should not be used to create production services due to the limitations outlined in this article. 
```

Comment:
I believe this is not strictly necessary... If people are willing to setup a prod database they have the full power to do so, but they also have a month to switch to be a paying customer

Reply:
These were all notes on the original article (intercom), but I guess that recommendation was in place in case you forgot to pay or something? I agree though, it's still a full service account, just a VM limitation .